meetings/2018-05-03.md

```diff
@@ -0,0 +1,148 @@
+
+### nodejs/build: Bots + Apps in the Org #1247
+
+- Has come up a few times recently (unfortunately not all stakeholders on call)
+- We want to use bots in the github org for one purpose or another (website redesign, i18n, etc)
+- Work was done previously around that, how do we go forward with this?
+- Can come down to “what priviledges does the bot need”? If it’s something we need to do outside of what we’ve normally done then need to explore. 
+- Some context missing today, we should follow up on that repo / issue
+- Lot’s of folk want to do things around this, so defining explicit documentation (or locating/updating the documentation) would be helpful
+- Permissions are definitely static, so a reference is needed
+- Tierney will create an issue in the admin repo and ask
+
+### nodejs/TSC: Wiki Management #525
+
+- The wiki was out of date; had stale information
+- Some information on nodejs.org, not great to have two different places for information
+- (may have also been triggered by some wiki vandalism)
+- Wiki may already be disabled and we’ll all converge on one spot: if you know of something useful pull it over
+- Might want to add an issue to the website redesign repo (or a link to some getting started content)
+
+### nodejs/admin: Travel Fund approval and being careful about funds available #99
+
+- Nothing new to report here
+- Might need to define a treasurer on the tsc side, Jem is still volunteering from 
+- Next step: PR into repo about “the how” the treasurer will proceed
+- The reason for a treasurer is to have someone who can provide the context, so having someone on the TSC side could be helpful
+- Position it: we’ve selected jem from commcomm to do the role. Let us know if there are concerns. If you want to be involved as well, let us know.
+- Interesting opportunity to invest in an “admin role” that could be more cross functional. Can be a shared thing.
+- Myles proposed some cross functional tsc / comm comm meetings, might be a good time to think about those again. (Good topic for those meetings: budget checkin, collab summit updates)
+
+### nodejs/community-committee: CommComm Membership: Call For Feedback on New Definitions #276
+
+- A lot of definitions have gone into a pr that has been broadly accepted.
+- There were no objections, so it was pushed to the board
+- There will still be other todos and PR’s that come out of this. (emeriti needs further detail, errata pr)
+- Probably other work that needs to happen around working groups, and empowering other groups (working groups)
+- When big changes happen from a sub committee, we document it in some sort of history file so we have a record. (HISTORY.md? Changelog?)
+
+### nodejs/mentorship: "Priority Document" outlining areas of greatest need for Node.js ecosystem
+
+- Is there a document about best things to work on for ecosystem
+- created a plan for what they though it should look like
+- https://github.com/nodejs/mentorship/issues/38#issuecomment-386313778
```

Comment:
Just https://github.com/nodejs/mentorship/issues/38 is fine. The comment linked to is irrelevant.

Reply:
Also, should be "what they thought* "